### PR TITLE
Update preferences.md to reflect v3 scheduler availability

### PR DESCRIPTION
**Pull request by AntoineQ1**
I updated the text to reflect the availability of v3 scheduler on all platforms.

_Don't hesitate to point out if this change needs to be phrased differently._

### CODE REVIEW

**Comment by dae** [comment]
Your change is fine, but given that we don't actually show that option in the preferences screen anymore, maybe we should just remove mention of it instead?

**Comment by AntoineQ1** [comment]
Yes, absolutely, I didn't think about that !

File: src/preferences.md [hunk 1]
```diff
@@ -106,6 +89,9 @@ Disable this option to hide the card count at the bottom of the screen.
 **Show next review time above answer buttons**\
 Useful to know how far in the future your cards are being pushed.
 
+**Spacebar (or enter) also answers card**\
+Defines whether a press on the space bar (or the enter key) also answers card.
```
Comment (dae) [inline src/preferences.md:93]:
```suggestion
Defines whether a press on the space bar (or the enter key) also answers cards.
```

**Comment by dae** [comment]
Thank you!

**Pull request merged**